src/labkey/Domain.ts

```diff
@@ -408,6 +408,58 @@ export function save(config: SaveDomainOptions): XMLHttpRequest {
     });
 }
 
+export interface ValidateDomainNameExpressionOptions extends CreateDomainOptions {
+    includeNamePreview: boolean
+}
+/**
+ * Validate name expression(s) for domain design.
+ */
+export function validateNameExpressions(config: ValidateDomainNameExpressionOptions): XMLHttpRequest {
+
+    let options: ValidateDomainNameExpressionOptions = arguments.length > 1 ? {
+        success: arguments[0],
+        failure: arguments[1],
+        domainDesign: arguments[2],
+        options: arguments[3],
+        kind: arguments[4],
+        includeNamePreview: arguments[5],
+    } : config;
+
+    return request({
+        url: buildURL('property', 'validateNameExpressions.api', options.containerPath),
+        method: 'POST',
+        success: getCallbackWrapper(getOnSuccess(options), options.scope),
+        failure: getCallbackWrapper(getOnFailure(options), options.scope, true),
```

Comment:
Is `scope` meant to be added to options when parsing the `arguments` array?  I guess this isn't done in other methods that use this pattern, but I'm unclear when the `arguments` option is used and how people know what order to provide those arguments.